LibraryApp/src/components/navigators.js

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react';
+import {StyleSheet} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import {booksList, libraryForm, showBookDetails} from 'constants/navigators';
+import {Global, Colors, Typography} from 'stylesheets';
+import {navigationRef} from 'helpers/navigation';
+import Form from 'components/form';
+import BooksList from 'components/books-list';
+import ShowBook from 'components/show-book';
+import HandleHardwareBack from 'components/shared/handle-hardware-back';
+
+const {textCenter} = Global;
+const {fs18, bold, uppercase} = Typography;
+const {lightBlue, purple} = Colors;
+
+const Stack = createStackNavigator();
+
+const Navigators = () => {
+  const {headingContainer, formLabel} = styles;
+  const [isInitialScreen, setIsInitialScreen] = useState(true);
+  const isCurrentScreenInitialOne = (state) => {
+    const route = state.routes[state.index];
+    if (route.state) {
+      return isCurrentScreenInitialOne(route.state);
+    }
+    return state.index === 0;
+  };
+
+  return (
+    <>
+      {isInitialScreen && <HandleHardwareBack />}
```

Comment:
looks complicated. Is it possible to move this into the component itself. As we are going to have a single initialScreen? Pls let me know your thoughts behind this

Reply:
Moved this to book-lists component.